handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: Restart Midonet Manager
+  service:
+    name: midonet-manager
```

Comment:
We ain't restarting 'midonet-manager' there is no such service, its an apache WEB app, so Apache needs to get resterted